packages/desktop-client/src/components/budget/MobileBudget.js

```diff
@@ -306,4 +306,4 @@ export default connect(
     initialBudgetMonth: state.app.budgetMonth,
   }),
   actions,
-)(withThemeColor(colors.p5)(BudgetWrapper));
+)(withThemeColor(colors.buttonPositiveBorder)(BudgetWrapper));
```

Comment:
I'm not entirely sure what the withThemeColor function actually did since I don't use Chrome. It only replaces the `content` color in `<meta name="theme-color" content="...">` tag in index.html. Looking this up, it is used by some mobile browsers to change the browsers URL bar color.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name/theme-color
With that in mind, I changed the withThemeColor to the sidebarBackground to match the header and footer.

Reply:
The different tabs of the mobile apps on `master` have different top bar colors, and this setting indeed updates the browser chrome to match (for example, the area around the notch on iPhones).

Reply:
Can you confirm it is fixed? I only use Firefox on mobile.

Reply:
It looks like the theme color is always white now, and the bars still have the dark appearance as opposed to the different colors they had on `master`.